book/website/reproducible-research/rdm/rdm-data-curation.md

```diff
@@ -0,0 +1,42 @@
+(rr-rdm-data-curation)=
+# Data Curation
+
+Data curation is one of the key aspects of Research Data Management (RDM). 
+It involves the capturing, appraisal, disposal, description, preservation, access, reuse and transformation of research data. 
+Data curation promotes the use of data from the point of creation to ensuring that its used for the purpose it is intended for. 
+Data curation also enables you to easily access data sets and information that you need in an organised format, you can easily extract data from whichever repository it is stored. 
+You can imagine that if you don't follow the data curation process, it becomes difficult to access that data.
+
+## Data Curation Pipeline
+
+### 1. Data capture
+Data capture is the process of gathering or collecting data from different sources, which will be processed and used for certain purposes. 
+These sources may be research articles in electronic format or databases like electronic databases among others. 
+This stage focuses on ensuring that the data captured is actually fit for purpose and ready for curation.
+
+### 2. Data appraisal
+Data appraisal and selection are fundamental stages in the data curation process. 
+In data appraisal, you are required to select the appropriate data by entering, digitizing, transcribing, checking, validating and cleaning the data; in addition to anonymizing the data where necessary, describing the data and managing and storing the data.  
+As you appraise the data you need to adhere to the documented guidance, policies and legal requirements in evaluating research data in a particular organization and selecting the data that requires long term preservation. 
+Your appraisal and selection policy should ensure consistency, transparency and accountable decision making.
+
+### 3. Data disposal 
+Data disposal involves disposing of data that has not be selected for retention. 
+You need to have an appraisal policy which will guide you on the data required for archiving, redeployment, transfer of custody or ownership or destroying the research data.
+
+### 4. Data description
+Data description requires that you are able to interpret the data; derive data; produce research outputs; author publications; data anonymisation; data visualization; data validation and prepare the data for preservation. 
+So it is important that you describe the research data so that it is discoverable and usable over time. 
+There are also  metadata standards that already exist to help you with standardised descriptions. 
+
+### 5. Data preservation
+Please find more information in the chapter on [Data Management Plans](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-dmp.html).
```

Comment:
Using the syntax here for linking across the book: https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html
```suggestion
Please find more information in the chapter on {ref}`Data Management Plans<rr-rdm-dmp>`.
```